src/app/payment/paymentMethod/MasterpassPaymentMethod.tsx

```diff
@@ -20,8 +20,17 @@ const MasterpassPaymentMethod: FunctionComponent<MasterpassPaymentMethodProps &
         },
     }), [initializePayment]);
 
+    const formatLocale = useCallback((localeLanguage: string) => {
```

Comment:
This looks more like a util method, can it be moved outside the component?

Reply:
I switch approach to consume a function that returns user's locale as an IETF language tag, based on all available sources.